src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -9,304 +9,20 @@ knotxVersions:
 ---
 ## Overview
 
-In this tutorial, we will set up a simple project based on the [Knot.x Starter Kit](https://github.com/Knotx/knotx-starter-kit) template.
+In this tutorial, we will set up a simple project based on the [Knot.x Starter Kit](https://github.com/Knotx/knotx-starter-kit) template, just as we did in [the previous tutorial](http://knotx.io/tutorials/getting-started-with-knotx-stack/2_0/), but this time we will use [Docker](https://www.docker.com/).
 
-We will implement a custom handler that will print a "Hello world from Knot.x!" message.
+This is the second part of *Getting started* tutorials, and completion of [Getting Started with Knot.x Stack tutorial](http://knotx.io/tutorials/getting-started-with-knotx-stack/2_0/) is **required** to fully understand this tutorial.
 
```

Comment:
What you’re going to learn:
- How to build a custom Docker image containing project-specific configs and extensions using Gradle
- How to validate the Docker image with functional / system tests

Reply:
done